homeassistant/components/tts/__init__.py

```diff
@@ -47,6 +47,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+TtsAudioType = Tuple[Optional[str], Optional[bytes]]
```

Comment:
Please use a future import and use the new style instead

Reply:
In alias it can't be used yet. Actually this kind of things is checked by CI pretty well, so if it passes, it's ok 🙂

Reply:
Oh right 🤦‍♂️